packages/vm/Cargo.toml

```diff
@@ -22,6 +22,8 @@ iterator = ["cosmwasm-std/iterator"]
 staking = ["cosmwasm-std/staking"]
 # this enables all stargate-related functionality, including the ibc entry points
 stargate = ["cosmwasm-std/stargate"]
+# extend stargate with ibcv3 extra functionality
+ibcv3 = ["cosmwasm-std/ibcv3", "stargate"]
```

Comment:
We don't need this feature in the VM as it is supposed to work for both cases. Also it means something different here that con cosmwasm-std. There is means: _only v3_ and here it means _v1 and v3_.

What we can do is simply:

stargate = ["cosmwasm-std/stargate", "cosmwasm-std/ibcv3"]

Reply:
There is an issue... a ibc3 contract will fail in an ibc1 runtime, as it will be missing the relayer field.

I *could* just make those fields `Option<Addr>` and then it is truly compatible and not need any flag at all. But then no contract could count on them being set. Such contracts that need version negotiation or relayer address will need ibcv3 runtime.

Reply:
Ah, nevermind... this is on VM and not STD. You are completely right.